blocks/hero-block/_index.scss

```diff
@@ -0,0 +1,77 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-hero {
+	display: grid;
+	grid-template-areas: "hero";
+	justify-content: center;
+
+	> * {
+		grid-area: hero;
+	}
+
+	&--stacked {
+		grid-template-areas:
+			"hero"
+			"info";
+
+		@include scss.block-properties("hero-stacked");
+		@include scss.block-components("hero-stacked");
+	}
+
+	&--light {
```

Comment:
can you add the dark variant as well? I noticed ``${BLOCK_CLASS_NAME}--dark`` but not corresponding scss class

Reply:
This has been added @JackHowa